Static analysis error in `label/kv.go`

Error from `nogo`:
```
io_opentelemetry_go_otel/label/kv.go:140:42: tautological condition: non-nil != nil
```

The start of the function returns immediately if `value == nil`, and then later on it checks `value != nil && err == nil` after calling `json.Marshal()`. I believe that second use of `value` should be a different symbol (maybe `b` ?).

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -37,6 +37,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fix `Code.UnmarshalJSON` to work with valid json only. (#1276)
 - The `resource.New()` method changes signature to support builtin attributes and functional options, including `telemetry.sdk.*` and
   `host.name` semantic conventions; the former method is renamed `resource.NewWithAttributes`. (#1235)
+- Fix condition in `label.Any`. (#1299)
 
 ### Removed
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -37,7 +37,6 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fix `Code.UnmarshalJSON` to work with valid json only. (#1276)
 - The `resource.New()` method changes signature to support builtin attributes and functional options, including `telemetry.sdk.*` and
   `host.name` semantic conventions; the former method is renamed `resource.NewWithAttributes`. (#1235)
-- Fix condition in `label.Any`. (#1299)
 
 ### Removed
 
@@ -48,6 +47,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Fixed
 
 - The `go.opentelemetry.io/otel/api/global` packages global TextMapPropagator now delegates functionality to a globally set delegate for all previously returned propagators. (#1258)
+- Fix condition in `label.Any`. (#1299)
 
 ## [0.13.0] - 2020-10-08
 

--- a/label/kv.go
+++ b/label/kv.go
@@ -137,7 +137,7 @@ func Any(k string, value interface{}) KeyValue {
 	case reflect.String:
 		return String(k, rv.String())
 	}
-	if b, err := json.Marshal(value); value != nil && err == nil {
+	if b, err := json.Marshal(value); b != nil && err == nil {
 		return String(k, string(b))
 	}
 	return String(k, fmt.Sprint(value))

